### PR TITLE
docs:add slack link

### PR DESCRIPTION
**Pull request by sunshinemingo**
add invite link of Slack into README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ Please check the [contributing guide](https://www.x-lab.info/github-analysis-rep
 
 Please check the [architect](https://www.x-lab.info/github-analysis-report-2020/) and [workflow](https://www.x-lab.info/github-analysis-report-2020/) if you want to better understand the project.
 
+## Communication
+
+Click here to join the [Slack Communication Group](https://join.slack.com/t/x-github-analysis/shared_invite/zt-gz8ykv3l-qP53IpajsZIGsx6dFo_8Lg) if you want to know more about the project and join it.
```
Comment (frank-zsy) [inline README.md:19]: Why use title case for `slack communication group`?

File: README.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ Please check the [contributing guide](https://www.x-lab.info/github-analysis-rep
 
 Please check the [architect](https://www.x-lab.info/github-analysis-report-2020/) and [workflow](https://www.x-lab.info/github-analysis-report-2020/) if you want to better understand the project.
 
+## Communication
+
+Click here to join the [slack communication group](https://join.slack.com/t/x-github-analysis/shared_invite/zt-gz8ykv3l-qP53IpajsZIGsx6dFo_8Lg) if you want to know more about the project and join it.
```
Comment (frank-zsy) [inline README.md:19]: Sorry, but I think `Slack` should be capitalized.

**Comment by sunshinemingo** [comment]
I have changed the title.

File: README.md (hunk 2 above)
Comment (frank-zsy) [reply README.md]: And if you use `Click here to join`, maybe the url should be added on `here` but not on `Slack communication group`

Comment (sunshinemingo) [reply README.md]: OK

**Pull request merged**